### DISCUSSION TITLE
Feeds should show all posts

### DISCUSSION THREAD

**Discussion by Zero3K** [bluesky-social/atproto#1413] [Protocol (atproto)]
It would be nice if feeds showed all posts instead of up to 7 days. I read a post from a user that was complaining about the limit.

**Comment by dholms** [comment]
This is a short term mitigation against the postgres query planner dropping us into very degraded paths. We'll be changing out timeline composition approach soon (to fanout on write) which will allow us to increase the date threshold

**Comment by Zero3K** [comment]
To how old? Or will it be a user-configurable setting?

**Comment by Zero3K** [comment]
How much longer until it is implemented? Search results are currently severely limited (I am only getting the past 10 minutes' worth of results when searching for Bluesky).
